jest.config.base.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   testPathIgnorePatterns: ["/node_modules/", "/dist/"],
   moduleFileExtensions: [...defaults.moduleFileExtensions, "ts", "tsx"],
   moduleNameMapper: {
-    "^(unmock|unmock-cli|unmock-core)(?:/dist)?((?:/.*)|$)":
+    "^(unmock|unmock-cli|unmock-core|unmock-jest)(?:/dist)?((?:/.*)|$)":
```

Comment:
No one:
My regex mind: We can stop editing this with `^(unmock(-[A-Za-z]+)?)(?:/dist)?((?:/.*)|$)`

Reply:
If it passes the build, we can assume the Lerna gods are appeased.